app/views/components/docs/banner.yml

```diff
@@ -0,0 +1,18 @@
+name: Banner
+description: A page banner, designed to highlight important information
+body: Passing an aside to the banner will mean the banner renders in a grid layout.
+accessibility_criteria: |
+  The banner must:
+
+  - be visually distinct from other content on the page
+  - meet WCAG AA standard for colour contrast (4.5:1 for normal text and 3:1 for large text)
```

Comment:
We've had trouble in the past interpreting what normal text and large text are. I think we should be explicit about one ratio for all text:

* all text must have a contrast ratio higher than 4.5:1 against the background colour to meet WCAG AA

For context, see the discussion here: https://github.com/alphagov/govuk_frontend_toolkit/issues/316